providers/atlassian/resources/atlassian_jira.go

```diff
@@ -0,0 +1,251 @@
+				"id":   llx.StringData(group.GroupID),
+				"name": llx.StringData(group.Name),
+			})
+		if err != nil {
+			log.Fatal().Err(err)
+		}
+		res = append(res, mqlAtlassianJiraUserGroup)
+	}
+	return res, nil
+}
+
+func (a *mqlAtlassianJira) serverInfos() (*mqlAtlassianJiraServerInfo, error) {
+	conn := a.MqlRuntime.Connection.(*jira.JiraConnection)
+	jira := conn.Client()
+	info, response, err := jira.Server.Info(context.Background())
+	if err != nil {
+		log.Fatal().Err(err)
+	}
+	if response.Status != "200 OK" {
+		log.Fatal().Msgf("Received response: %s\n", response.Status)
+	}
+	res, err := CreateResource(a.MqlRuntime, "atlassian.jira.serverInfo",
+		map[string]*llx.RawData{
+			"baseUrl":        llx.StringData(info.BaseURL),
+			"serverTitle":    llx.StringData(info.ServerTitle),
+			"buildNumber":    llx.IntData(int64(info.BuildNumber)),
+			"deploymentType": llx.StringData(info.DeploymentType),
+		})
+	return res.(*mqlAtlassianJiraServerInfo), err
+}
+
+func (a *mqlAtlassianJira) projects() ([]interface{}, error) {
+	conn := a.MqlRuntime.Connection.(*jira.JiraConnection)
+	jira := conn.Client()
+	projects, response, err := jira.Project.Search(context.Background(), nil, 0, 1000)
+	if err != nil {
+		log.Fatal().Err(err)
+	}
+	if response.Status != "200 OK" {
+		log.Fatal().Msgf("Received response: %s\n", response.Status)
+	}
+
+	res := []interface{}{}
+	for _, project := range projects.Values {
+		mqlAtlassianJiraProject, err := CreateResource(a.MqlRuntime, "atlassian.jira.project",
+			map[string]*llx.RawData{
+				"id":       llx.StringData(project.ID),
+				"name":     llx.StringData(project.Name),
+				"uuid":     llx.StringData(project.UUID),
+				"key":      llx.StringData(project.Key),
+				"url":      llx.StringData(project.URL),
+				"email":    llx.StringData(project.Email),
+				"private":  llx.BoolData(project.IsPrivate),
+				"deleted":  llx.BoolData(project.Deleted),
+				"archived": llx.BoolData(project.Archived),
+			})
+		if err != nil {
+			log.Fatal().Err(err)
+		}
+		res = append(res, mqlAtlassianJiraProject)
+	}
+	return res, nil
+}
+
+func (a *mqlAtlassianJira) issues() ([]interface{}, error) {
+	conn := a.MqlRuntime.Connection.(*jira.JiraConnection)
+	jira := conn.Client()
+	validate := ""
+	jql := "order by created DESC"
+	fields := []string{"status", "project"}
+	expands := []string{"changelog", "renderedFields", "names", "schema", "transitions", "operations", "editmeta"}
+	issues, response, err := jira.Issue.Search.Get(context.Background(), jql, fields, expands, 0, 1000, validate)
+	if err != nil {
+		log.Fatal().Err(err)
+	}
+	if response.Status != "200 OK" {
+		log.Fatal().Msgf("Received response: %s\n", response.Status)
+	}
+
+	res := []interface{}{}
+	for _, issue := range issues.Issues {
+		mqlAtlassianJiraIssue, err := CreateResource(a.MqlRuntime, "atlassian.jira.issue",
+			map[string]*llx.RawData{
+				"id":      llx.StringData(issue.ID),
+				"project": llx.StringData(issue.Fields.Project.Name),
```

Comment:
i guess we can get more than this data. At least the name/description would be nice

Reply:
Added the description, I actually can't find a field name or title or anything similar